docs/config.md

```diff
@@ -1706,6 +1724,10 @@ not being able to discover that you have the objects that you have. If you want
 to have this disabled and keep the network aware of what you have, you must
 manually announce your content periodically.
 
+> [!CAUTION]
+> Setting `Reprovider.Interval=0` will only disable the periodic reprovides.
+> New CIDs will stil lbe announced. To disable both, set [`Provider.Enabled=false`](#providerenabled) as well.
```

Comment:
```suggestion
> New CIDs will still be announced. To disable both, set [`Provider.Enabled=false`](#providerenabled) as well.
```

Reply:
Good catch, I'll update this section, it is outdated.